src/cmdline/mod.rs

```diff
@@ -337,6 +337,18 @@ impl From<Cmdline> for Vec<u8> {
     }
 }
 
+impl fmt::Debug for Cmdline {
```

Comment:
Why not use derive?

Reply:
I liked this format better, but I've added a derive now.